ansible/inventory/local/group_vars/docker-deploy.yml

```diff
@@ -14,3 +14,4 @@ shortcut_name: "Shadow_Hand_Launcher" # Specify the name for the desktop icon (d
 optoforce: false # Specify if optoforce sensors are going to be used with a branch name (Only for Flexible Hand)
 launch_hand: false # Specify if hand driver should start when double clicking desktop icon
 demo_icons: true # Generates desktop icons to run demos
+run_when_docker_launches: "terminator -x bash -c '/usr/local/bin/setup_dexterous_hand.sh && bash || bash'" #what to run when the Docker launches
```

Comment:
no commands in ansible parameters. Find more elegant way please :)

Reply:
Fixed in the latest commit